internal/gatewayapi/testdata/envoyproxy-tls-settings-invalid-ns.out.yaml

```diff
@@ -310,7 +286,6 @@ xdsIR:
               - ECDSA-SECP256R1-SHA256
               sni: example.com
               useSystemTrustStore: true
-            weight: 1
```

Comment:
The route will not be added if it is a TCP/UDP route and contains invalid configurations

Reply:
@alexwo - let's maybe update the PR description and title so that the release notes will reflect the two behaviors:
- 500 for HTTP
- connection reset for TCP/UDP

In terms of UX, users experience a reset in this case right? Just want to make sure that the connections on the client side don't hang and wait for some timeout, so that it's clear to users that the request is explicitly rejected.

Reply:
If the route associated with the TLS connection is removed, ongoing connections that were relying on this route will be dropped. This will most likely result in a TCP reset (RST) or a termination of the TLS session, effectively severing the connection.